lib/Controller/ChatController.php

```diff
@@ -432,20 +432,15 @@ public function receiveMessages(int $lookIntoFuture,
 			return $response;
 		}
 
+		$comments = $this->filterHistorySince($comments, $attendee);
```

Comment:
NEeds to go before `if (empty($comments)) {` so you return if there is nothing left

Reply:
right